movieapp/src/components/MovieListErrorBoundary/MovieListErrorBoundary.tsx

```diff
@@ -5,8 +5,7 @@ type ErrorProps = {
 };
 
 export function MovieListErrorBoundary(props: ErrorProps): JSX.Element {
```

Comment:
Why did u call it with prefix? MovieList*??

it's regular component and it can wrap any other component to handle errors. 

good practice to wrap each Route into the boundary

Reply:
fixed